### PR TITLE
Added responsive logo

### PR DESCRIPTION
**Pull request by bezi19**
responsive logo, user avatar, colors change and register text change

### CODE REVIEW

File: frontend/src/components/pages/Layout.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ import {
   faTasks,
 } from "@fortawesome/free-solid-svg-icons";
 import logoUrl from "../../assets/img/logo/longlogo.svg";
+import logoUrl2 from "../../assets/img/logo/logo.svg"
```
Comment (kaspher) [inline frontend/src/components/pages/Layout.js:13]: zmień to może na coś w stulu LogoUrlMobile?

File: frontend/src/assets/css/colors.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-$primary-color: #228b22;
-$secondary-color: #000;
-$background-color: #fff;
-$hover-background-color: #228b22;
+$primary-color: #386641;
```
Comment (kaspher) [inline frontend/src/assets/css/colors.scss:1]: dodaj to do `tailwind.config.js`

File: frontend/src/components/pages/Layout.js [hunk 3]
```diff
@@ -104,7 +106,7 @@ const Layout = () => {
                     </span>
                   )}
                   <img
-                    src={user ? cloudFrontDomainName + user.id : logoUrl}
+                    src={user ? cloudFrontDomainName + user.id : userAvatar}
```
Comment (kaspher) [inline frontend/src/components/pages/Layout.js:109]: wywal to userAvatar, default bedzie brany z `aws s3`

File: frontend/src/components/pages/auth/Register.js [hunk 4]
```diff
@@ -118,9 +118,10 @@ function Register() {
             Zarejestruj się
           </button>
         </form>
-        <div className="mt-6 text-blue-500 text-center">
-          <Link to="/login" className="hover:underline">
-            Masz już konto? Zaloguj się tutaj!
+        <div className="mt-6 text-center">
+        Masz już konto?{" "}
+          <Link to="/login" className="text-blue-500 hover:underline">
+            Zaloguj się tutaj!
```
Comment (kaspher) [inline frontend/src/components/pages/auth/Register.js:124]: 👍🏽

**Review by kaspher — CHANGES_REQUESTED** [review]
🔥

**Comment by bezi19** [comment]
fixed

**Pull request merged**